doc/reference/indexer.rst

```diff
@@ -58,19 +58,58 @@ Watching
 
 File watchers are used to keep the index up-to-date.
 
-The type of watcher used depends on your system. Currently the following
-watchers will be used, in order of priority:
+Several watching systems can be used, by default Phpactor will choose the
+first supported one:
 
-1. ``inotifywait``: **Linux** only, react immediately[1] to file changes.
+watchman
+~~~~~~~~
 
-2. ``find``: **Linux/Mac/POSIX** Poll the system for changes every 5 seconds.
+**Linux/Mac** cross platform, reacts immediately to file changes, see Watchman_ documentation.
 
-3. ``php``: **Any system**: Poll system using PHP (slow) every 5 seconds.
+Watchman is the recommended watcher.
 
-To contribute to the available watchers see `amp-fswatch
-<https://github.com/phpactor/amp-fswatch>`_.
+Installation:
 
-If you want to find out which watcher your system is using, enable _`logging`.
+.. tabs::
+
+    .. tab:: Debian/Ubuntu
+       
+        .. code-block:: bash
+
+            apt install watchman
```

Comment:
You might want to add the version number from which watchman is included in the repositories.
On my side:
```sh
$ lsb_release -a
LSB Version:    core-9.20170808ubuntu1-noarch:printing-9.20170808ubuntu1-noarch:security-9.20170808ubuntu1-noarch
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.5 LTS
Release:        18.04
Codename:       bionic
```

I have to install it manually.